### ISSUE TITLE
Failed to synchronize registry data from server, closing connection.

### ISSUE THREAD

**Issue by JennyBlock** [NillerMedDild/Enigmatica6#2179]
Hi!

I've been playing Enigmatica 6 for around two months or so together with my boyfriend. He hosts a LAN and I join with a direct connection. This went smoothly until the most recent update to 0.4.10, where I discovered that I could no longer join his world. I get the following error: "Failed to synchronize registry data from server, closing connection.". Below is part of the log file directly after connecting to the LAN world. 

[22:20:43] [Render thread/INFO]: Built PerkTree with 417 perks!
[22:20:46] [Render thread/INFO]: Injecting existing registry data into this CLIENT instance
[22:20:46] [Render thread/INFO]: Registry Block: Found a missing id from the world jaopca:storage_blocks.aeternium
[22:20:46] [Render thread/INFO]: Registry Block: Found a missing id from the world jaopca:storage_blocks.terminite
[22:20:46] [Render thread/INFO]: Registry Block: Found a missing id from the world jaopca:storage_blocks.thallasium
[22:20:46] [Render thread/INFO]: Registry Block: Found a missing id from the world supplementaries:flag
[22:20:46] [Render thread/INFO]: Registry Block: Found a missing id from the world undergarden:potted_ditchbulb
[22:20:46] [Render thread/ERROR]: Unidentified mapping from registry minecraft:block
	jaopca:storage_blocks.aeternium: 11965
	jaopca:storage_blocks.terminite: 11966
	jaopca:storage_blocks.thallasium: 11967
	supplementaries:flag: 1200
	undergarden:potted_ditchbulb: 8175

[22:20:46] [Render thread/ERROR]: Missing registry data for network connection:
	minecraft:block: jaopca:storage_blocks.aeternium
	minecraft:block: jaopca:storage_blocks.terminite
	minecraft:block: jaopca:storage_blocks.thallasium
	minecraft:block: supplementaries:flag
	minecraft:block: undergarden:potted_ditchbulb

[22:20:46] [Netty Client IO #1/ERROR]: Failed to load registry, closing connection.
[22:20:46] [Netty Client IO #1/ERROR]: Connection closed, not continuing handshake

I mainly feel like either me or him are missing files however we have the same clean install?
(FYI, we use Curseforge to update to the most recent versions and already both tried uninstalling and reinstalling the 0.4.10/0.4.11 versions to no avail) 

Please let me know if any of you have a solution to this problem. If I need to include more files or info, I would be willing to do so ofcourse! :)

**Comment by NillerMedDild** [comment]
> I mainly feel like either me or him are missing files however we have the same clean install?

You're right, that's what it looks like. Perhaps both of you could update to 0.4.12 and see how that goes?

**Comment by JennyBlock** [comment]
So we updated to 0.4.12 but we're still getting the same error. I just let my boyfriend check his latest.log and this happens on his side:
[22:49:23] [Render thread/INFO]: Started serving on 61050
[22:49:23] [Render thread/INFO]: [CHAT] Local game hosted on port 61050
[22:49:31] [Netty Server IO #3/INFO]: Skipping config sync, No mods have registered a syncable config

Not sure if this has anything to do with it however.

**Comment by NillerMedDild** [comment]
That seems irrelevant tbh. Can I have both you and your boyfriend's latest.log please?

**Comment by JennyBlock** [comment]
Here you go :)
Mine: [latest.log](https://github.com/NillerMedDild/Enigmatica6/files/6488223/latest.log)
My boyfriends: [latest.log](https://github.com/NillerMedDild/Enigmatica6/files/6488227/latest.log)

**Comment by NillerMedDild** [comment]
Seems to me like something is going wrong with updating, a few things seem to be stuck on older values. Maybe try clean reinstalls for both of you, otherwise you may have to make or join a server :/ on the positive side, you'll get much better performance on a server.

**Comment by JennyBlock** [comment]
Hi! Thanks for the support :)

In the end I found another method that worked for us.
My boyfriend installed NBTExplorer. In each save world folder is a level.dat\FML\Registries\minecraft:blocks path, where we just deleted the entries/IDs that gave the error messages. Then started the world again, I could join and everything worked smoothly again. 

By the way, thanks for making and updating this awesome modpack!

Kind regards,
Jennifer :)
